docs/services/vision/segmentation.md

```diff
@@ -218,6 +219,83 @@ The following parameters are available for a `detector_3d_segmenter`.
 
 Click **Save config** and proceed to [test your segmenter](#test-your-segmenter).
 
+## Configure an `obstacles_depth` segmenter
+
+This segmenter model is for depth cameras, and is best for motion planning with transient obstacles.
+If you have [configured the frame system](/services/frame-system/#configuration) for the components of your robot, then you can implement this model to identify well separated objects above a flat plane.
+
+To configure an `obstacles_depth` segmenter, first decide if you want to use the segmenter with or without intrinsic parameters.
+
+This is determined by the value of the configuration attribute `return_pcds`:
+
+- `true`: this segmenter will return point clouds within the `GeometryInFrame` object it captures.
+- `false`: this segmenter will return a single point within the `GeometryInFrame` object it captures.
+
+If `true`, start configuring your segmenter by following [these instructions](/services/frame-system/#configuration) to configure the relative spatial orientation of the components of your robot within Viam's [frame system service](/services/frame-system/).
+
+After doing this, your camera will populate its own `Properties` with the intrinsic parameters of the geometric configuration.
+You can then get those parameters from your camera through the [camera API](/components/camera/#getproperties).
+
+Then, configure an `obstacles_depth` segmenter:
```

Comment:
```suggestion
To configure your segmented you need the Properties parameters from your camera which you can get with the [camera API](/components/camera/#getproperties).

Then, configure your `obstacles_depth` segmenter:
```

a -> your : because at this point we've given so much info it feels more specific

Reply:
Makes sense but not sure about needing properties here-- @kharijarrett if I implement this suggestion do you think line 237 makes sense/should be included?